Pod/Classes/VOKCoreDataManager.m

```diff
@@ -93,7 +95,7 @@ - (NSManagedObjectContext *)tempManagedObjectContext
         return nil;
     }
 
-    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSConfinementConcurrencyType];
+    NSManagedObjectContext *tempManagedObjectContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
```

Comment:
Probably. I'll have to do a bit more research on this before I can officially figure out this bit.